config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile += ['payment.js']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.